src/shipit_pulse_listener/shipit_pulse_listener/listener.py

```diff
@@ -1,120 +1,100 @@
-from datetime import datetime, timedelta
-from taskcluster.utils import slugId
-from cli_common.pulse import create_consumer, run_consumer
+from cli_common.pulse import run_consumer
 from cli_common.taskcluster import TaskclusterClient
 from cli_common.log import get_logger
-import copy
-import json
-import re
+from shipit_pulse_listener.hook import Hook
+import asyncio
 
 logger = get_logger(__name__)
 
-HOOK_REGEX = re.compile(r'([\w\-_]+):([\w\-_]+)')
 
-
-class PulseListener(object):
+class HookStaticAnalysis(Hook):
     """
-    Listen to pulse messages and trigger new tasks
+    Taskcluster hook handling the static analysis
     """
-    def __init__(self, secrets_path, client_id=None, access_token=None):
-        self.taskcluster = TaskclusterClient(client_id, access_token)
-        self.tasks = []
-
-        # Fetch pulse credentials from TC secrets
-        logger.info('Using secrets', path=secrets_path)
-        required = ('PULSE_USER', 'PULSE_PASSWORD', 'PULSE_QUEUE')
-        secrets = self.taskcluster.get_secrets(secrets_path, required)
-
-        # Use pulse consumer from bot_common
-        self.consumer = create_consumer(
-            secrets['PULSE_USER'],
-            secrets['PULSE_PASSWORD'],
-            secrets['PULSE_QUEUE'],
-            secrets.get('PULSE_TOPIC', '#'),
-            self.got_message
+    def __init__(self, branch):
+        super().__init__(
+            'project-releng',
+            'services-{}-shipit-static-analysis-bot'.format(branch),
+            'exchange/mozreview/',
+            'mozreview.review.published',
```

Comment:
Nit: Shouldn't this be [mozreview.commits.published](https://mozilla-version-control-tools.readthedocs.io/en/latest/mozreview/notifications.html#mozreview-commits-published) instead?